docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
         deploy:
             resources:
                 limits:
-                    cpus: ${MAX_CORES-/1}
+                    cpus: ${MAX_CORES-1.0}
```

Comment:
add a comment explaining how this syntax works (provide a link to a website explaining variable expansion with default values)

Reply:
Good, then add a comment in the `yml` file